MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -244,24 +248,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     // Item click listener
-    ItemClickSupport.addTo(recyclerView).setOnItemClickListener(new ItemClickSupport.OnItemClickListener() {
-      @Override
-      public void onItemClicked(RecyclerView recyclerView, int position, View view) {
-        ExampleItemModel model = adapter.getItemAt(position);
-
-        // in case it's an info tile
-        if (model != null) {
-          if (showJavaExamples) {
-            startActivity(model.getJavaActivity());
-          } else {
-            startActivity(model.getKotlinActivity());
-          }
-
-          analytics.clickedOnIndividualExample(getString(model.getTitle()), loggedIn);
-          analytics.viewedScreen(getString(model.getTitle()), loggedIn);
-        }
-      }
-    });
+    ItemClickSupport.addTo(recyclerView).setOnItemClickListener(this);
```

Comment:
If the click listener is added as part of `onResume`, this call is redundant.

Do lifecycle callbacks work well to resolve the issue?

Reply:
I know I'd moved it to `onResume` but forgot to push my continued work, haha.

Reply:
When I removed the time measurement stuff, the activity would still be called twice when the activity launching is in `onResume()`.

Reply:
I just pushed the work I'd mentioned above. When I tap once normally on an example, my following logs come into the logcat

![Screen Shot 2019-10-04 at 4 58 48 PM](https://user-images.githubusercontent.com/4394910/66246307-3fd0b180-e6c8-11e9-944d-3251ca9d3d52.png)

But when I click twice on the example, the 2nd `onItemClicked` fires after `onPause()`
![Screen Shot 2019-10-04 at 5 02 08 PM](https://user-images.githubusercontent.com/4394910/66246427-c4bbcb00-e6c8-11e9-8557-f4b48c96ec04.png)